# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344984952 25200
#      Tue Aug 14 15:55:52 2012 -0700
# Node ID f3a5338aaa0122944fe393d25f6f51bf5b3f0abd
# Parent  8593bed236f119816da3136dc8710d22905ff00d
build: Remove unneeded -Ioctave-value, -Ioperators from AM_CPPFLAGS.

* src/Makefile.am: Remove unneeded -Ioctave-value, -Ioperators from AM_CPPFLAGS.

diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -21,19 +21,19 @@
 include $(top_srcdir)/build-aux/common.mk
 
 AUTOMAKE_OPTIONS = subdir-objects
 
 ## Search local directories before those specified by the user.
 AM_CPPFLAGS = \
   -I$(top_srcdir)/libcruft/misc \
   -I../liboctave -I$(top_srcdir)/liboctave \
-  -Ioctave-value -I$(srcdir)/octave-value \
+  -I$(srcdir)/octave-value \
+  -I$(srcdir)/operators \
   -Iparse-tree -I$(srcdir)/parse-tree \
-  -Ioperators -I$(srcdir)/operators \
   -Iinterp-core -I$(srcdir)/interp-core \
   -Iinterpfcn -I$(srcdir)/interpfcn \
   -Icorefcn \
   -I$(top_builddir)/libgnu -I$(top_srcdir)/libgnu
 
 octlib_LTLIBRARIES = liboctinterp.la
 
 if AMCOND_BUILD_COMPILED_AUX_PROGRAMS
